src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -477,12 +469,18 @@ const Notebooks = _.flow(
               this.refresh()
             }
           }),
-          !!creatingGalaxy && h(NewGalaxyModal, {
-            isOpen: creatingGalaxy,
+          //todo if status is deleting consider not opening the panel
+          !!openGalaxyConfigDrawer && h(NewGalaxyModal, {
+            isOpen: openGalaxyConfigDrawer,
             namespace,
-            app: apps,
-            onDismiss: () => this.setState({ creatingGalaxy: false }),
-            onSuccess: () => { this.setState({ creatingGalaxy: false }) }
+            apps,
+            onDismiss: () => {
+              this.setState({ openGalaxyConfigDrawer: false })
+              this.refresh()
```

Comment:
This should have been in the onSuccess() however even when there it doesn't appear to be updating correctly. Need to revisit this and how to update the button once the creation has begun